NakamaTest/Source/NakamaTest/NakamaTest.Build.cs

```diff
@@ -9,7 +9,7 @@ public class NakamaTest : ModuleRules
 	public NakamaTest(ReadOnlyTargetRules Target) : base(Target)
 	{
 		PCHUsage = PCHUsageMode.UseExplicitOrSharedPCHs;
-		PrivateDependencyModuleNames.AddRange(new string[] { "Core", "CoreUObject", "Engine", "InputCore", "NakamaCore"});
+		PrivateDependencyModuleNames.AddRange(new string[] { "Core", "CoreUObject", "Engine", "InputCore", "NakamaCore", "WebSockets" });
```

Comment:
builtin web sockets module used to be  _super_ buggy: https://github.com/EpicGames/UnrealEngine/pull/7267  and we avoided using it

Reply:
We still don't use it as implementation, but I found warnings on defines used from there without proper module referencing. I can see what those are, make sure that are actually needed, and clean up this.